scripts/provisioning/s3_setup

```diff
@@ -83,6 +83,7 @@ update_cluster_id=false
 is_rabbitmq_clustered=false
 cleanup_accounts=false
 cleanup_dependencies=false
+cleanup_both=false
```

Comment:
cleanup_both is not good variable name.. use something which is more informative, like: "cleanup_ldap_accounts_and_dependencies"

Reply:
Ajinkya suggested to use "cleanup" only. I'll go with that as it is short.